app.py

```diff
@@ -39,43 +40,55 @@
 
 # Setup Logging
 logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"),
-                    format='%(levelname)s: %(message)s')
+                    format='[%(asctime)s] %(levelname)s: %(message)s',
+                    datefmt='%m/%d/%y %H:%M:%S')
```

Comment:
you don't have to actually submit a `datefmt` parameter just cause you're using `asctime`. it defaults to ISO-8601 standard (https://docs.python.org/2/howto/logging.html#displaying-the-date-time-in-messages) which is `datefmt='%m/%d/%Y %I:%M:%S %p'`, and just slightly different from what you're doing. Of course, if you have a particular reason for doing it your way, that's cool, too.

Reply:
Short answer is I like military time in my logs, long answer is I played around with the format quite a bit and didn't realized I settled on something so close to the default in the end.